### PR TITLE
vuln: update core generator to filter active release lines

### PR DESCRIPTION
**Pull request by RafaelGSS**
There's no need to include old patches in the index.json. This PR filter only active release lines.

### CODE REVIEW

File: tools/create_index/index_generator.js [hunk 1]
```diff
@@ -1,36 +1,51 @@
...
-var vuln = {}
+let vuln = {}
 
 const createIndex = function (vulnDirectoryPath) {
   const files = fs.readdirSync(vulnDirectoryPath)
   getVulnDirectoryContents(files, vulnDirectoryPath)
   writeIndex(vuln, vulnDirectoryPath)
 }
 
+const supportedVersions = ['14.0.0', '16.0.0', '18.0.0', '19.0.0']
```
Comment (mhdawson) [inline tools/create_index/index_generator.js:13]: Is there some way to avoid hard coding this?

Comment (RafaelGSS) [reply tools/create_index/index_generator.js:13]: Easily, I don't think so.  @nodejs/releasers do we have an API with this info?

**Review by vdeturckheim — APPROVED** [review]

File: tools/create_index/index_generator.js (hunk 1 above)
Comment (RafaelGSS) [reply tools/create_index/index_generator.js:13]: I'll merge it. In case someone has a better idea, I can create a follow-up PR.

Comment (richardlau) [reply tools/create_index/index_generator.js:13]: > Easily, I don't think so. @nodejs/releasers do we have an API with this info?

https://github.com/pkgjs/nv with the `supported` alias.

**Pull request merged**